web/frontend/src/components/slam-2d-render.vue

```diff
@@ -243,12 +183,7 @@ const createAxisHelper = (name: string, rotation: number): THREE.AxesHelper => {
 };
 
 // create the background gray grid
-const createGridHelper = (
-  points: THREE.Points<THREE.BufferGeometry, THREE.Material | THREE.Material[]>
-): THREE.GridHelper => {
-  points.geometry.computeBoundingBox();
-
-  const boundingBox = points.geometry.boundingBox!;
+const createGridHelper = (boundingBox: THREE.Box3): THREE.GridHelper => {
```

Comment:
`computeBoundingBox()` was a strange side effect for this function. It has been removed and this function now simply takes any bounding box.